### PR TITLE
MPP-3475 - New strings for profile page

### PR DESCRIPTION
**Pull request by lloan**
Strings for [MPP-3475 PR](https://github.com/mozilla/fx-private-relay/pull/4033)

These only affect the profile page and replace existing strings.

### CODE REVIEW

**Review by peiying2 — APPROVED** [review]

**Comment by peiying2** [comment]
@flodolo please take another look when you get a chance.

File: en/profile.ftl [hunk 1]
```diff
@@ -117,9 +117,13 @@ profile-promo-email-blocking-option-none = None
 profile-promo-email-blocking-description-all-2 = { -brand-name-relay } is blocking all emails sent to this mask.
 profile-promo-email-blocking-description-promotionals = { -brand-name-relay } will attempt to block promotional emails while still forwarding emails like receipts and shipping information.
 profile-promo-email-blocking-description-none-2 = { -brand-name-relay } is not blocking any emails for this mask.
+profile-promo-email-blocking-description-none-3 = All emails sent to this mask will be forwarded to your real address.
 profile-promo-email-blocking-label-promotionals = Block promotions
+profile-promo-email-blocking-label-promotionals-2 = Blocking promo emails
```
Comment (flodolo) [inline en/profile.ftl:122]: I don't understand this.

Please add a comment that this is a about blocking promotional emails.

**Review by flodolo — COMMENTED** [review]
@lloan 
There is no need to flag a specific person for review, that's why we have the alias flagged automatically ;-)

> These only affect the profile page and replace existing strings.

If this is replacing strings, why are we not removing the replaced strings? Who's going to remove them and when?

**Comment by lloan** [comment]
> @lloan There is no need to flag a specific person for review, that's why we have the alias flagged automatically ;-)
> 
> > These only affect the profile page and replace existing strings.
> 
> If this is replacing strings, why are we not removing the replaced strings? Who's going to remove them and when?

Hi @flodolo, because I'm not very familiar with the whole process. Does this mean I can delete the old string from the .ftl file when submitting new strings that replace them? I've seen instances where we add a comment as "deprecated" above it, but not sure when either should happen. More than happy to remove them in this PR if that's the appropriate course. Thanks!

**Comment by flodolo** [comment]
If a string is unused it needs to be removed. When it's removed it depends on your processes: do you need to push updates to production using the old strings, or the next push to production will only use the new ones?

Marking as deprecated is OK, but someone needs to take ownership of cleaning them up regularly, otherwise a new locale is going to translate a lot of useless content.

File: en/profile.ftl (hunk 1 above)
Comment (lloan) [reply en/profile.ftl:125]: added

**Review by peiying2 — APPROVED** [review]

**Pull request merged**